### ISSUE TITLE
Form Verification and validation

### ISSUE THREAD

**Issue by Lexxville** [toocool/tripexpress#36]
It seems like a user can submit an empty form during the booking process, the form isn't processing passenger details rather it submits empty or null values to the database.

**Comment by toocool** [comment]
@Lexxville yeah I am aware about that, it's unfinished and I have pre-maturely pushed it
